packages/textlint/src/config/config-loader.ts

```diff
@@ -2,6 +2,8 @@ import { TextLintModuleResolver } from "../engine/textlint-module-resolver";
 
 // LICENSE : MIT
 "use strict";
+const fs = require("fs");
```

Comment:
It is better in TypeScript.
`@types/node` support Node.js Core module typing.
```ts
import * as fs from "fs"
```
It make it typing.